views/group/show.pug

```diff
@@ -2,31 +2,91 @@ extends ../layouts/main
 
 block content
   .group-show
-    h1.title.is-1= group.name
-    span.dates #{format(group.startDate, DATE_FORMAT)} - #{format(group.endDate, DATE_FORMAT)}
-    if group.subject
-      h2.is-size-4 Tárgy: #{group.subject}
-    h3.is-size-5 Szint: #{group.room}
-    h3.description= group.description
-    if !group.doNotDisturb
-      h2.is-size-4 Résztvevők
-      if group.users
-        ul
-          each user in group.users
-            li
+    .columns
+      .column
+        h1.title= group.name
+      .column.is-narrow
+        if isOwner
+          button.button.is-danger(type="button", onclick=`deleteGroup(${group.id})`) 
+            i.fas.fa-trash-alt
+            span &nbsp;Törlés
+
+    - const dateTimeStart = format(group.startDate, DATE_FORMAT).split(" ")
+    - const dateTimeEnd = format(group.endDate, DATE_FORMAT).split(" ")
+
+    table.table.is-fullwidth
+      thead
+        p.menu-label Adatok
+      tbody
+        tr
+          td
+            span.icon.has-text-info
+              i.fas.fa-fw.fa-building
+            span &nbsp;#{group.room}. szint
+          td
+            if group.doNotDisturb
+              span.icon.has-text-info
+                i.fas.fa-fw.fa-lock
+              span.has-tooltip-bottom(data-tooltip="A szervező letiltotta a csatlakozást") &nbsp;Zárt
+                sup ?
+            else
+              span.icon.has-text-info
+                i.fas.fa-fw.fa-unlock
+              span.has-tooltip-bottom(data-tooltip="Szabad a csatlakozás") &nbsp;Nyílt
+                sup ?
+        tr
+          if dateTimeStart[0] == dateTimeEnd[0]
```

Comment:
Instead of indexing like this, you can use array destructuring in line #14 and 15, like this:
`const [startDate, startTime] = format(group.startDate, DATE_FORMAT).split(' ')`